ext/dapr-ext-workflow/dapr/ext/workflow/dapr_workflow_context.py

```diff
@@ -49,26 +55,33 @@ def is_replaying(self) -> bool:
         return self.__obj.is_replaying
 
     def create_timer(self, fire_at: Union[datetime, timedelta]) -> task.Task:
+        self._logger.debug(f'instance_id {self.instance_id} create_timer: {fire_at}')
         return self.__obj.create_timer(fire_at)
 
     def call_activity(self, activity: Callable[[WorkflowActivityContext, TInput], TOutput], *,
                       input: TInput = None) -> task.Task[TOutput]:
+        self._logger.debug(f'instance_id {self.instance_id} call_activity: {activity.__name__}')
         return self.__obj.call_activity(activity=activity.__name__, input=input)
 
     def call_child_workflow(self, workflow: Workflow, *,
                             input: Optional[TInput],
                             instance_id: Optional[str]) -> task.Task[TOutput]:
+        self._logger.debug(f'instance_id {self.instance_id} \
+                           call_child_workflow: {workflow.__name__}')
+
         def wf(ctx: task.OrchestrationContext, inp: TInput):
-            daprWfContext = DaprWorkflowContext(ctx)
+            daprWfContext = DaprWorkflowContext(ctx, self._logger.get_options())
             return workflow(daprWfContext, inp)
         # copy workflow name so durabletask.worker can find the orchestrator in its registry
         wf.__name__ = workflow.__name__
         return self.__obj.call_sub_orchestrator(wf, input=input, instance_id=instance_id)
 
     def wait_for_external_event(self, name: str) -> task.Task:
+        self._logger.debug(f'instance_id {self.instance_id} wait_for_external_event: {name}')
```

Comment:
if we are going to follow a k:v approach, I would prefer to use a utility method with a particular format. The other option is we have full sentences in logs similar to the other logs in python-sdk.

Reply:
The formatter is added as part of `LoggerOptions`. It's set to `TIME SCOPE LOG_LEVEL: LOG_MESSAGE` by default.

Reply:
but then you are formatting the message as `k:v` pairs ... if so would suggest to add a simple utility method for that

Reply:
Moved to sentences